app/src/main/res/drawable/back_arrow.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="19dp"
+    android:height="19dp"
+    android:viewportWidth="19"
+    android:viewportHeight="19">
+  <path
+      android:pathData="M13.158,2.367C13.066,2.275 12.956,2.202 12.836,2.152C12.716,2.102 12.587,2.076 12.457,2.076C12.327,2.076 12.198,2.102 12.078,2.152C11.957,2.202 11.848,2.275 11.756,2.367L5.178,8.946C5.104,9.019 5.046,9.106 5.006,9.202C4.966,9.297 4.946,9.4 4.946,9.504C4.946,9.607 4.966,9.71 5.006,9.806C5.046,9.902 5.104,9.989 5.178,10.062L11.756,16.641C12.144,17.029 12.77,17.029 13.158,16.641C13.545,16.253 13.545,15.627 13.158,15.239L7.426,9.5L13.165,3.76C13.545,3.38 13.545,2.747 13.158,2.367Z"
+      android:strokeWidth="0.2"
+      android:fillColor="#1E232C"
```

Comment:
light/dark mode implementasyonu için bu renkleri @color/someColor şeklinde ayarlamanız faydalı olacaktır.
İleride dark mode'da ekleyecektiniz, öyle hatırlıyorum.

Reply:
Dark mode'u eklemek şu an kesin ve önceliğimiz değil, bunu da zamanımız kalırsa yapacağız